installers/pgo-installer/bin/install.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+INSTALLNS="pgo-install"
+INSTALLDIR="$PGOROOT/installers/pgo-installer"
+
+kubectl create ns $INSTALLNS
```

Comment:
Do  we want to assume that `kubectl` is on the `$PATH`?

Reply:
I may end up removing this as well. I was using for initial testing before i added the roles.

Reply:
looking back - we were assuming that that `kubectl` and `oc` are on the `$PATH` for the current ansible install. If that needs to be changed it shouldn't be difficult

Reply:
Similarly, one option for `oc` is to first see if the `kubectl` commands work correctly on a OCP install.

They should be compatible: [https://docs.openshift.com/container-platform/3.10/cli_reference/differences_oc_kubectl.html(https://docs.openshift.com/container-platform/3.10/cli_reference/differences_oc_kubectl.html)

Reply:
I tested the installer with only `kubectl` installed before I added `oc`. I didn't notice any issues with the install when using `kubectl` with openshift.

Looking at the differences shown in the link, most things shouldn't be necessary. The login feature won't be an issue because the pod will be using the `kubernetes_in_cluster` option and won't need to login. As long as we don't need access to the OpehShift resources (DeploymentConfigs, BuildConfigs, Routes, ImageStreams, and ImageStreamTags) I don't see an issue only using `kubectl`.

Reply:
The only one I could see us potentially needing are `ImageStreams`, but I'd advise waiting until that becomes a request. So for now let's ship `kubectl` and simplify the dependency management.

Reply:
I've removed the `oc` steps from the centos7 image. For rhel7/ubi7 i'm installing with a common package that installs both so i may just leave that unless there is a reason to switch back to adding the repo file for kubernetes.